src/pages/instances/InstanceVga.tsx

```diff
@@ -17,36 +17,43 @@ declare global {
 }
 
 interface Props {
-  controlTarget?: HTMLSpanElement | null;
+  onMount: (handler: () => void) => void;
+  onFailure: (message: string, e: unknown) => void;
+  inTabNotification: Notification | null;
 }
 
-const InstanceVga: FC<Props> = ({ controlTarget }) => {
-  const notify = useNotify();
+const InstanceVga: FC<Props> = ({ onMount, onFailure, inTabNotification }) => {
```

Comment:
Now that we call it "Graphic" in the frontend under the console tab, shall we rename this to `InstanceGraphicConsole`?